sdcm/cluster.py

```diff
@@ -2016,23 +2015,28 @@ def scylla_setup(self, disks):
-        if self.parent_cluster.params.get('workaround_kernel_bug_for_iotune'):
-            self.log.warning(dedent("""
-                Kernel version is {}. Due to known kernel bug in this version using predefined iotune.
-                related issue: https://github.com/scylladb/scylla/issues/5181
-                known kernel bug will cause scylla_io_setup fails in executing iotune.
-                The kernel bug doesn't occur all the time, so we can get some succeed gce instance.
-                the config files are copied from a succeed GCE instance (same instance type, same test
-            """.format(self.kernel_version)))
-            self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} --no-io-setup {}'
-                             .format(devname, ','.join(disks), extra_setup_args))
-            for conf in ['io.conf', 'io_properties.yaml']:
-                self.remoter.send_files(src=os.path.join('./configurations/', conf),  # pylint: disable=not-callable
-                                        dst='/tmp/')
-                self.remoter.run('sudo mv /tmp/{0} /etc/scylla.d/{0}'.format(conf))
-        else:
-            self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} {}'
-                             .format(devname, ','.join(disks), extra_setup_args))
+
+        with self._get_backtraces_lock:
+            if self.parent_cluster.params.get('workaround_kernel_bug_for_iotune'):
+                self.log.warning(dedent("""
+                    Kernel version is {}. Due to known kernel bug in this version using predefined iotune.
+                    related issue: https://github.com/scylladb/scylla/issues/5181
+                    known kernel bug will cause scylla_io_setup fails in executing iotune.
+                    The kernel bug doesn't occur all the time, so we can get some succeed gce instance.
+                    the config files are copied from a succeed GCE instance (same instance type, same test
+                """.format(self.kernel_version)))
+                self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} --no-io-setup {}'
+                                 .format(devname, ','.join(disks), extra_setup_args))
+                for conf in ['io.conf', 'io_properties.yaml']:
+                    self.remoter.send_files(src=os.path.join('./configurations/', conf),  # pylint: disable=not-callable
+                                            dst='/tmp/')
+                    self.remoter.run('sudo mv /tmp/{0} /etc/scylla.d/{0}'.format(conf))
+            else:
+                self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} {}'
+                                 .format(devname, ','.join(disks), extra_setup_args))
+
+            # Remove bash core which generated by scylla_setup for internal test purpose.
+            self.remoter.run("sudo rm /var/lib/systemd/coredump/core.bash.*")
```

Comment:
add `ignore_status=True` since the core may not be there...

Reply:
done